client/my-sites/plans-comparison/plans-comparison.tsx

```diff
@@ -47,169 +56,171 @@ const getRowMobileLayout = ( breakpoint: number, pageClass: string ) => `
 `;
 
 export const globalOverrides = css`
-	#content.layout__content {
```

Comment:
Most of the changes in `globalOverrides` are just indentation adjustments after moving almost everything (except for tooltip adjustments) under the `.is-sections-plans`.